bundles/org.openhab.binding.meater/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,156 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="meater"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="meaterapi">
+		<label>Meater Cloud REST API</label>
+		<description>This bridge represents the Meater cloud REST API connector</description>
+
+		<config-description>
+			<parameter name="email" type="text" required="true">
+				<label>Email</label>
+				<description>The email used to login to Meater cloud account</description>
+			</parameter>
+			<parameter name="password" type="text" required="true">
+				<label>Password</label>
+				<context>password</context>
+				<description>The password used to login to Meater cloud account</description>
+			</parameter>
+			<parameter name="refresh" type="integer" min="10" unit="s">
+				<label>Refresh Interval</label>
+				<description>Specifies the refresh interval in seconds</description>
+				<default>30</default>
+			</parameter>
+		</config-description>
+	</bridge-type>
+
+	<thing-type id="meaterprobe">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="meaterapi"/>
+		</supported-bridge-type-refs>
+
+		<label>Meater Probe</label>
+		<description>This thing represents a Meater probe</description>
+
+		<channels>
+			<channel id="internalTemperature" typeId="internalTemperature"/>
+			<channel id="ambientTemperature" typeId="ambientTemperature"/>
+			<channel id="cookId" typeId="cookId"/>
+			<channel id="cookName" typeId="cookName"/>
+			<channel id="cookState" typeId="cookState"/>
+			<channel id="cookTargetTemperature" typeId="cookTargetTemperature"/>
+			<channel id="cookPeakTemperature" typeId="cookPeakTemperature"/>
+			<channel id="cookElapsedTime" typeId="cookElapsedTime"/>
+			<channel id="cookRemainingTime" typeId="cookRemainingTime"/>
+			<channel id="cookEstimatedEndTime" typeId="cookEstimatedEndTime"/>
+			<channel id="status" typeId="status"/>
+			<channel id="lastConnection" typeId="lastConnection"/>
+		</channels>
+
+		<properties>
+			<property name="vendor">Apption Labs</property>
+			<property name="model">Meater</property>
+		</properties>
+
+		<representation-property>deviceId</representation-property>
+
+		<config-description>
+			<parameter name="deviceId" type="text" required="true">
+				<label>Device Id</label>
+				<description>Unique id for your Meater probe</description>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+	<channel-type id="status">
+		<item-type>String</item-type>
+		<label>Current Status</label>
+		<description>Information on current status</description>
+		<state readOnly="true" pattern="%s"/>
+	</channel-type>
+
+	<channel-type id="lastConnection">
```

Comment:
You could add category `Time` on all your things type with item type `Number:Time` or `DateTime`.

Reply:
Fixed!